src/index.html

```diff
@@ -1,13 +1,253 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./img/icons/icon-logo.svg"
+      sizes="32x32"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body id="home">
+    <header class="header">
+      <div class="header__row">
+        <div class="header__top">
+          <a href="#">
+            <img
+              src="img/header/header-logo.png"
+              alt="logo"
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/137a09bb-a1d9-4a6e-9c15-ddcebe3a91e0)
Fix this in all code.